.github/global.yml

```diff
@@ -230,3 +230,14 @@
   description: related to producing reproducible results
   color: "86C579"
   aliases: [reproducibility]
+
+# Documentation
+- name: documentation::testing
+  description: related to writing tests
+  color: "31A354"
+- name: documentation::developer
+  description: related to API and internal workings of project
+  color: "31A354"
+- name: documentation::user
+  description: related to documentation describing how the project is used
+  color: "31A354"
```

Comment:
To encourage a better documentation structure, we should stay close to the [Diátaxis framework](https://diataxis.fr) as a system framework for technical writing, since it focuses on different user needs instead of separating people into groups. E.g. an issue from a user looking for guidance for how to use conda is about the quality or existence of a how-to in the docs, and not specific about whether it's a "user" or a "developer" (they are often the same). We should organize our issues accordingly, since they'll allow us to see gaps in us catering to these different user needs.

```suggestion

# Documentation
- name: documentation::tutorials
  description: related to tutorials (learning-oriented)
  color: "31A354"
- name: documentation::how-tos
  description: related to how-tos (task-oriented)
  color: "31A354"
- name: documentation::reference
  description: related to the API and internals (information-oriented)
  color: "31A354"
- name: documentation::explanation
  description: related to higher-level clarification (understanding-oriented)
  color: "31A354"
```